docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8u151-jdk-alpine3.7
```

Comment:
Any specific reason to use this tag? It's pretty outdated. I think that `openjdk:8u272-slim` could be an alternative.